modules/kn-trigger-describe.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * serverless/event_workflows/serverless-kn-trigger.adoc
+// * /serverless/event_workflows/serverless-using-brokers.adoc
```

Comment:
Not critical, but there's an unnecessary `/` at the beginning of this file path. Same in a few other files in this PR.

Reply:
Cool, I'll fix these as I come across them in files. Thank you!